src/int.rs

```diff
@@ -0,0 +1,167 @@
+//! Stack-allocated big signed integers.
+
+use num_traits::Zero;
+use subtle::{Choice, ConditionallySelectable, ConstantTimeEq};
+
+use crate::{Limb, Uint};
+
+mod add;
+mod cmp;
+mod div;
+mod mul;
+mod neg;
+mod sub;
+
+#[derive(Copy, Clone, Hash, Debug, PartialEq)]
+pub struct Int<const LIMBS: usize> {
+    // TODO(erik): replace with Choice once I've figured how to const-construct those.
```

Comment:
no problem here, worst case scenario, just use u8, CtChoice is more or less that.